---
layout: subsite-galaxy
website: https://virology.usegalaxy.eu
---

# Welcome to Galaxy for Virology

![Virology Galaxy](/assets/media/virology_workbench.jpeg){:.rna-intro-right}

On the Virology Galaxy workbench you can get, process, analyze, and visualize data. 

1. TOC
{:toc}

We offer **workflows and training material specific to SARS-CoV-2 and Avian Influenza Virus (AIV)** (more viruses to come soon). These resources will help you efficiently and effectively analyze and interpret your data.

# Get started

Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take
[a guided tour]({{ page.website }}/tours/core.galaxy_ui){:target="_blank"} through Galaxy's user interface.


# Tutorials

The tutorials below cover topics in virology concerning AIV and SARS-CoV-2. These and more tutorials are available in the [Galaxy Training Network (GTN)](https://training.galaxyproject.org/).

  |  | Slides | Hands-on | Recording | Input dataset | Workflows 
--- | --- | --- | --- | --- | --- 
**Introduction** |  |  |  |  |
Welcome and introduction to Galaxy | [<i class="fa fa-slideshare" aria-hidden="true"></i>](https://training.galaxyproject.org/topics/introduction/tutorials/galaxy-intro-short/slides.html){:target="_blank"} | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-short/tutorial.html){:target="_blank"} |[<i class="fa fa-video-camera" aria-hidden="true"></i>](https://training.galaxyproject.org/videos/watch.html?v=introduction/tutorials/galaxy-intro-short/slides){:target="_blank"}| [<i class="fa fa-files-o" aria-hidden="true"></i>](https://doi.org/10.5281/zenodo.582600){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-short/workflows/){:target="_blank"} 
**Avian Influenza Virus** |  |  |  |  |
Avian influenza viral strain analysis from gene segment sequencing data | [Topic Overview <i class="fa fa-slideshare" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/variant-analysis/slides/introduction.html){:target="_blank"} | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/aiv-analysis/tutorial.html){:target="_blank"} |  |  |
**SARS-CoV-2** |  |  |  |  |
SRA Aligned Read Format to Speed Up SARS-CoV-2 data Analysis|[<i class="fa fa-slideshare" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/ncbi-sarf/slides.html){:target="_blank"}|[<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/ncbi-sarf/tutorial.html){:target="_blank"}|[Tutorial <i class="fa fa-video-camera" aria-hidden="true"></i>](https://gallantries.github.io/video-library/videos/galaxy-interface/ncbi-sarf/tutorial/){:target="_blank"} / [Lecture <i class="fa fa-video-camera" aria-hidden="true"></i>](https://gallantries.github.io/video-library/videos/galaxy-interface/ncbi-sarf/slides/){:target="_blank"}||
Mutation calling, viral genome reconstruction and lineage/clade assignment from SARS-CoV-2 sequencing data|[Topic Overview <i class="fa fa-slideshare" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/variant-analysis/slides/introduction.html){:target="_blank"}|[<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.html){:target="_blank"} |[<i class="fa fa-video-camera" aria-hidden="true"></i>](https://gallantries.github.io/video-library/videos/variant-analysis/sars-cov-2-variant-discovery/tutorial){:target="_blank"}|[<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/5036687){:target="_blank"}|[<i class="fa fa-share-alt" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/workflows/){:target="_blank"}
Unicycler assembly of SARS-CoV-2 genome with preprocessing to remove human genome reads| [<i class="fa fa-slideshare" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/assembly/tutorials/assembly-with-preprocessing/slides.html){:target="_blank"}|[<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/assembly/tutorials/assembly-with-preprocessing/tutorial.html){:target="_blank"}|[<i class="fa fa-video-camera" aria-hidden="true"></i>](https://gallantries.github.io/video-library/videos/assembly/assembly-with-preprocessing){:target="_blank"}|[<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/3732359){:target="_blank"}|[<i class="fa fa-share-alt" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/assembly/tutorials/assembly-with-preprocessing/workflows/){:target="_blank"}
Virtual screening of the SARS-CoV-2 main protease with rxDock and pose scoring||[<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/computational-chemistry/tutorials/covid19-docking/tutorial.html){:target="_blank"}|  |[<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/3730474){:target="_blank"}|[<i class="fa fa-share-alt" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/computational-chemistry/tutorials/covid19-docking/workflows/){:target="_blank"}
From NCBI's Sequence Read Archive (SRA) to Galaxy: SARS-CoV-2 variant analysis|[Topic Overview <i class="fa fa-slideshare" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/variant-analysis/slides/introduction.html){:target="_blank"}|[<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/sars-cov-2/tutorial.html){:target="_blank"}|  |[<i class="fa fa-files-o" aria-hidden="true"></i>](http://zenodo.org/record/3906454){:target="_blank"}|[<i class="fa fa-share-alt" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/sars-cov-2/workflows/){:target="_blank"}
Removal of human reads from SARS-CoV-2 sequencing data|  |[<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/human-reads-removal/tutorial.html){:target="_blank"}|[<i class="fa fa-video-camera" aria-hidden="true"></i>](https://gallantries.github.io/video-library/videos/sequence-analysis/human-reads-removal/tutorial/){:target="_blank"}|  |[<i class="fa fa-share-alt" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/human-reads-removal/workflows/){:target="_blank"}
{:.table.table-striped}

You find more in-depth material and information about SARS-CoV-2 on the dedicated COVID-19 project site [galaxyproject.org/projects/covid19](https://galaxyproject.org/projects/covid19/){:target="_blank"}.

# Tools

There is a variety of tools related to virology integrated in Galaxy. This selection is developed for analyses with viral genome data.

TOOL | DESCRIPTION
--- | ---
**Cojac** | Tool package to analyse co-occurrence of mutations on amplicons.
**Freyja** | Recover relative lineage abundances from SARS-CoV-2 samples from a sequencing dataset.
**ivar** | Tool package for viral amplicon-based sequencing.
**Nextclade** | Viral genome clade assignment, mutation calling, alignment, sequence quality checks and phylogenetic placement.
**Pangolin** | Phylogenetic assignment of outbreak lineages: to assign a SARS-CoV-2 genome sequence the most likely lineage based on the PANGO nomenclature system.
**Read It and Keep** | Rapid decontamination of SARS-CoV-2 sequencing reads.
**snipit** | Tool to summarise SNPs relative to a reference sequence.
**VAPOR** | Classification of Influenza samples from raw short read sequence data for downstream analysis.
{:.table.table-striped}


# Workflows

To orchestrate tools and help users with their analyses, several workflows are available. They formally arrange tools in a defined order and with defined but customizable parameters.

## Avian Influenza Virus
![Avian Influenza Virus Strain Analysis Worfklow Diagram](/assets/media/virology_aiv_wf_landscape.png)

Workflow: [Avian influenza viral strain analysis from gene segment sequencing data](https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/aiv-analysis/tutorial.html){:target="_blank"}

This workflow uses a collection of reference sequences, per segment, and a Illumina sequenced paired-end sample data to subtype the given sample. In three major steps after the first quality checks, the eight segments of the Influenza A virus are aligned to the reference genome and the subtype is determined: 

**1. Variant Analysis:** Find the best matches between the sequencing data and the reference sequences of each segment. This gives a preliminary subtype and based on the reference collection, gives the best reference to map the sequencing data to <i>for each segment</i>. Mapping to the hybrid reference is performed using BWA-MEM.

**2. Consensus Sequence Construction:** The mapping to the best reference is used to construct a consensus sequence for each segment. This is done using the ivar consensus tool.

**3. Phylogeny:** The consensus sequences are aligned using MAFFT and one phylogenetic tree per segment is constructed using IQ-Tree. The trees can be used to determine related strains of the given sample.


## SARS-CoV-2
Available workflows about SARS-CoV-2 are:
1. Variant Calling
- SARS-CoV-2 PE/SE Illumina ARTIC/WGS variant calling
- SARS-CoV-2 ONT ARTIC variant calling
- SARS-CoV-2 PE Illumina ARTIC ivar analysis
2. Consensus from Variation
3. Variation Reporting

Find further links and descriptions about the workflows on [galaxyproject.org/projects/covid19/workflows](https://galaxyproject.org/projects/covid19/workflows/){:target="_blank"}.

## Newcastle Disease Virus
<i>coming soon</i>

## Lumpy Skin Disease Virus
<i>coming soon</i>


# Cooperation

![Virology ZODIAC Project](/assets/media/virology_zodiac.png){:.rna-intro-left}

The development of the tools and material is a collaborative effort of the Galaxy community and **ZO**onotic **DI**sease Integrated **AC**tion Initiative (ZODIAC) project members. ZODIAC is an International Atomic Energy Agency (IAEA) funded project to detect and prevent zoonotic outbreaks worldwide. Find out more about the 5-year-project on the [ZODIAC project website](https://nucleus.iaea.org/sites/zodiac/SitePages/Home.aspx).

<br />
<br />
<br />
<br />
